mods/d2k/sequences/vehicles.yaml

```diff
@@ -60,10 +60,138 @@ trike:
 		Facings: -32
 		Remap: 54F94B
 	muzzle:
-		Filename: DATA.R16
-		Start: 3996
+		Combine:
+			0:
+				Filename: DATA.R16
+				Frames: 3996, 38, 3996, 38, 3996, 38, 3996, 38
```

Comment:
Actually to match base D2K why do we need to have 8 frames? Isn't the animation using 7 there?

Reply:
Fixed